portfolio/src/main/webapp/index.jsp

```diff
@@ -81,7 +81,7 @@
           </div>
           <div class="container">
             <div class="text">
-              <form method="POST" enctype="multipart/form-data" action="<%= uploadUrl %>">
+              <form id="comment-form" method="POST" enctype="multipart/form-data" action="<%= uploadUrl %>">
```

Comment:
Let's add a css rule to make this and the login-form below "display: none" by default, so that they won't show up if your login request takes some time to resolve.

Reply:
Got it!